runner/Svarog.cs

```diff
@@ -255,6 +255,7 @@ public void Run(string[] args)
             commandLine.WithParsed(options => m_PresentationLayer?.Create(options));
 
             RunScript(@"dofile ""scripts\\Library.lua""");
+            RunScript(@"dofile ""scripts\\ItemLibrary.lua""");
```

Comment:
This should be done from lua, maybe from the Library.lua file, not from engine code